Cargo.toml

```diff
@@ -25,7 +25,7 @@ clap = { version = "4.0.11", features = ["derive"] }
 clap_complete = "4.0.2"
 inferno = { version = "0.11.0", default_features = false, features = ["multithreaded", "nameattr"] }
 opener = "0.6.0"
-shlex = "1.1.0"
+shlex = "1.3.0"
```

Comment:
Please remove this -- the requirements have not changed and I've updated the lockfile to use 1.3.0 already.